src/components/app-content-renderer/styles/panels.scss

```diff
@@ -20,56 +19,151 @@
+                padding-bottom: var(--pf-global--spacer--md);
+                overflow: hidden;
+                text-overflow: ellipsis;
+                white-space: nowrap;
+                &::after {
+                  content: '\00a0';
+                }
+              }
+            }
+          }
+          .estate-title {
+            .pf-c-description-list__text {
+              width: 140px;
+              p {
+                white-space: nowrap; 
+                overflow: hidden;
+                text-overflow: ellipsis;
+              }
+            }
           }
-          font-size: var(--pf-global--FontSize--xl) !important;
-          width: 100%;
-        }
-        .estate-count {
-          font-size: var(--pf-global--FontSize--xl) !important;
         }
-        .estate-title {
-          font-size: var(--pf-global--FontSize--sm) !important;
+      }
+    }
+
+    @media only screen and (max-width: 1199px) {
+      background-size: cover;
+      .pf-c-description-list {
+        grid-template-columns: repeat(calc(100% - var(--pf-c-description-list--ColumnGap)));
+        row-gap: 0;
+        .estate-group {
+          display: flex;
+          flex-wrap: wrap;
+          .estate-section {
+            >.pf-c-description-list__text {
+              p {
+                font-size: var(--pf-global--FontSize--md);
+                margin-bottom: var(--pf-global--spacer--md);
+                margin-top: var(--pf-global--spacer--md);
+              }
+              &::after {
+                content: '';
+              }
+              p:empty {
+                margin: 0;
+              }
+            }
+            font-size: var(--pf-global--FontSize--xl) !important;
+            width: 100%;
+          }
+          .estate-count {
+            h5 {
+              font-size: var(--pf-global--FontSize--md) !important;
+              padding-right: var(--pf-global--spacer--sm);
+            }
+          }
+          .estate-title {
+            p {
+              font-size: var(--pf-global--FontSize--md) !important;
+            }
+          }
         }
       }
     }
   }
 
+  .second-panel {
+    background-color: var(--pf-global--BackgroundColor--100);
+    color: var(--pf-global--Color--100);
+
+    @media screen and (min-width: 1450px) {
+      .pf-l-grid {
+        max-width: 1160px;
+        // grid-template-columns: repeat(auto-fill, calc(100% / 3 - var(--pf-c-description-list--ColumnGap)));
+      }
+    }
+
+    h1 {
+      font-size: var(--pf-global--FontSize--xl);
+      padding-bottom: var(--pf-global--spacer--lg);
+    }
+
+    h5 {
+      font-size: var(--pf-global--FontSize--sm);
+      font-weight:  300;
+    }
+
+    .whole-row {
+      width: 100%;
+      grid-template-columns: 1fr fit-content(20%)
+    }
+
+    .whole-row:not(:last-child) {
+      margin-bottom: 16px;
+    }
+
+    .custom-grid {
+      display: grid;
+      grid-template-columns: 1fr fit-content(100%);
+    }
 
+    .group-icon {
```

Comment:
I don't see this classname used

Reply:
@Hyperkid123 I almost never used the pre-defined classnames. I find it much easier to track the styling in a single file. Especially, when overriding for other break points.

Reply:
> Looks good. Can we use the pre-defined pf class names for margin and padding? In most cases, we don't need custom rules here. These `var(--pf-global--spacer--xl)` are equivalent to `pf-u-m/p(side)-(size)` classnames.

@Hyperkid123 I almost never used the pre-defined classnames. I find it much easier to track the styling in a single file. Especially, when overriding for other break points.

Reply:
> I don't see this classname used

@Hyperkid123 I'll look at it when I start working on the second panel..

Reply:
I understand that, but the thing is, we should try and use the design system to its maximum capacity. Sure, it won't be enough for everything, but it works with the basics. Yeah, it's just a couple of lines but when we add it up across the whole platform, we suddenly have hundreds, maybe thousands of lines of duplicate CSS.

Reply:
I think there are definitely points to be made on both sides for this. I would agree that we should use the design system to its maximum capacity and there are definitely benefits for using utility classes. However, I think in this specific scenario, they're going to cause more harm than good considering most of this stuff is custom and we're not following a lot of patternfly standards (like layouts and breakpoints).

For other applications that are more standard, like RBAC, Inventory, etc, I think moving towards using these utility classes is a good idea, and for new apps, let's give it a shot to see how it works.

Lets not worry about utility classes vs css a week before this gets pushed out to prod beta and instead just worry about getting it out the door. Eric is already using utility classes in a few places and I'm trusting his judgement to know when we should use them and when we shouldn't. If that's something we want to revisit post summit, that's fine. Let's not block a PR because people have different opinions on how to consume CSS because this argument is as old as time.